### ISSUE TITLE
Color Contrast

### ISSUE THREAD

**Issue by seanforyou23** [patternfly/patternfly-org#1222]
We're so close to being 100% free of a11y violations on the new doc site, let's investigate what changes are needed.

I think it's possible to clear the violations without sacrificing really any of the awesomeness of the design intent, but I do imagine we'll have to get creative and do some blending of the concepts. Please let me know how I can help!

<img width="1664" alt="Screen Shot 2019-06-04 at 6 30 36 PM" src="https://user-images.githubusercontent.com/5942899/59074742-e2079680-889a-11e9-959a-341b75d51aeb.png">

Here's a link to [axe](https://www.deque.com/axe/) where you can find a nice browser extension to help identify the problems.

**Comment by mmenestr** [comment]
Document to document issues: https://docs.google.com/document/d/1KVAeD1MCelLda9BUVnEZT3yeh7N62F9KKKRJC504JjE/edit

**Comment by christiemolloy** [comment]
@mmenestr do you have bandwidth to start working on the issues on the homepage (cc: @gdoyle1 ) when you're done, let me know and @jenny-s51 / @lareplog will implement it

**Comment by mmenestr** [comment]
Gina and I just set up a meeting to go over them on Friday! will keep you posted

**Comment by rachael-phillips** [comment]
Hi @christiemolloy @seanforyou23 @gdoyle1 @mmenestr this list is awesome and really encouraging that we are knocking these out!  Has this list been prioritized at all? https://docs.google.com/document/d/1KVAeD1MCelLda9BUVnEZT3yeh7N62F9KKKRJC504JjE/edit

**Comment by christiemolloy** [comment]
@rachael-phillips I have consolidated this list into github issues with the tags "p1" and "accessibility" and they are ready to be assigned. @mmenestr and @gdoyle1 are working on the color contrast issues on the homepage.
